doc/main/tbb_userguide/attach_flow_graph_to_arena.rst

```diff
@@ -0,0 +1,68 @@
+.. _attach_flow_graph_to_arena:
+
+Attach Flow Graph to an Arbitrary Task Arena
+======================
+
+
+|short_name| ``task_arena`` interface provides mechanisms to guide tasks execution within
+the arena by setting the preferred computation units or restricting part of computation units.
+In some case, you may want to use mechanisms within a flow graph.
```

Comment:
Is it necessary to have an 's' at the end?
```suggestion
In some cases, you may want to use mechanisms within a flow graph.
```

Reply:
Right